_ml-commons-plugin/algorithms.md

```diff
@@ -0,0 +1,220 @@
+---
+layout: default
+title: Supported Algorithms 
+has_children: false
+nav_order: 100
+---
+
+# Supported Algorithms
+
+ML Commons supports various algorithms to help train and predict ML models or test data-driven predictions without a model. This page outlines the algorithms supported by the ML Commons plugin and the API actions they support
+
+## Common limitation
+
+Except for the Localization algorithm, all of the following algorithms can only support retrieving 10,000 documents from an index as an input.
+
+## K-Means
+
+K-Means is a simple and popular unsupervised clustering ML algorithm. K-Means will randomly choose centroids, then calculate iteratively to optimize the position of the centroids until each observation belongs to the cluster with the nearest mean.
```

Comment:
We build K-Means on top of Tribuo library too, can you add some statement like Linear Regression ?